TUM Campus App/CardViewController.swift

```diff
@@ -109,38 +87,101 @@ extension CardViewController {
             self.tableView.tableHeaderView = search?.searchBar
         }
     }
+    
+}
+
+extension CardViewController: EditCardsViewControllerDelegate {
+    
+    func refresh(_ sender: AnyObject?) {
+        manager?.loadCards(skipCache: sender != nil).onSuccess(in: .main) { categories in
+            // TODO:
+            self.cards = categories.flatMap{ $0.elements.first }
```

Comment:
I thought you wanted them in categories?

Reply:
This is just a temp fix since I’m not yet updating the old cards but rather implementing new cards.

Reply:
This seems quite hacky. Your table view should be going off of the categories. My suggestion. We remove `getCellIdentifier` altogether and move that to the card key in the categories.

We'd still have some work to do with the other views afterwards but it's better than a halfway solution

Reply:
I agree.